akka-docs/rst/scala/code/docs/dispatcher/DispatcherDocSpec.scala

```diff
@@ -78,12 +78,8 @@ object DispatcherDocSpec {
       executor = "thread-pool-executor"
       # Configuration for the thread pool
       thread-pool-executor {
-        # minimum number of threads to cap factor-based core number to
-        core-pool-size-min = 2
-        # No of core threads ... ceil(available processors * factor)
-        core-pool-size-factor = 2.0
-        # maximum number of threads to cap factor-based number to
-        core-pool-size-max = 10
+        // or in Akka 2.4.2+
+        fixed-pool-size = 16
```

Comment:
Oh, I assumed there was general agreeing on it since the ticket is triaged. But you are right, maybe make the "simple" one the default and "here is a more flexible way to configure the number of threads based on the number of cores" for the original one? 

I'd guess the rationale for the original ticket was that people get the three values wrong all the time and misunderstand that it boils down to a single number @ktoso ?